fairseq/models/speech_to_speech/s2s_transformer.py

```diff
@@ -414,8 +416,8 @@ def forward(
         return decoder_out
 
 
-@register_model("s2spect_transformer")
```

Comment:
Be mindful about backward compatibility. We chose to name the model with s2spec on purpose, since it's clearer than what "Translatotron" means, and our implementation is not exactly the same as the original Translatotron model, which was LSTM-based anyway.